src/Test/Utilities/Desktop/TestResource.resx

```diff
@@ -123,7 +123,7 @@
 #pragma warning disable 414, 3021
```

Comment:
Instead of having this in a .resx file and having to escape XML entities it would be better to extract the source to a separate .cs file and include it in the test project as an embedded resource.